Documentation/ApiOverview/Database/Connection/Index.rst

```diff
@@ -95,19 +95,28 @@ bulkInsert()
    $connection = GeneralUtility::makeInstance(ConnectionPool::class)
       ->getConnectionForTable('sys_log')
    $connection->bulkInsert(
-      'sys_log',
-      [
-         [(int)$userId, (string)$details1],
-         [(int)$userId, (string)$details2],
-      ],
-      [
-         'userid',
-         'details',
-      ],
-      [
-         Connection::PARAM_INT,
-         Connection::PARAM_STR,
-      ]
+       'sys_log',
+       [
+           [
+               (int)$userId,
+               (string)$details1
+           ],
+           [
+               'userId' => (int)$userId,
+               'detail' => (string)$details2
+           ],
+       ],
+       [
+           // 'userid' does not exist in both records, so Doctrine uses value at array key position 0 of data parameter.
```

Comment:
i don't really get what this comment is trying to convey? You mean that there is one input array lacking the key 'userid' ?